scrapers_next/ma/people.py

```diff
@@ -3,7 +3,7 @@
 
 
 def list_url():
-    general_court_num = "192"
+    general_court_num = "193"
```

Comment:
Would also be good to change it in the example_source for LegDetail on line 13.

(Side note: I tried to run it with the updated value, but I'm getting an SSLCertVerificationError. Is it working for you locally?)

Reply:
Yeah I am getting a cert error for both the 192 and 193 sessions. I think MA made some changes. There will be another jira ticket to get this issue sorted out. The URL is the correct one we need though.